README.md

```diff
@@ -7,6 +7,111 @@ The SDK is compatible with:
 * .NET Framework 4.6.1
 * Mono
 
+# Installation
+## Using Nuget Package Manager
+ 1. Right-click on your project in the Solution Explorer and choose **Manage Nuget Packages...**
+ 2. Search for Stormpath. Install the `Okta.Sdk` package.
+
+## Using The Nuget Package Manager Console
+Simply run `install-package Okta.Sdk`. Done!
+
+# Client Configuration
+You can configure the `OktaClient` in one of three ways:
+
+With a `okta.yaml` file (either in the root of the project, or in the current user's "home" folder).
```

Comment:
The home folder thing needs a little more explanation. We could break it out into a bulleted list:

* The root of the project
* A `.okta` folder in the current user's home folder (`~/.okta/okta.yml` on *nix machines, `%userprofile%\.okta\okta.yml` on Windows)